deployment/helmfile.yaml

```diff
@@ -14,3 +24,7 @@ releases:
       - name: universalnode
       - namespace: laos-bigbang
       - imageTag: {{ requiredEnv "TAG" }}
+      - contractAddress: {{ .Values.contractAddress }}
```

Comment:
I'd replace the lines from `contractAddress` to `startingBlock` with `- {{ toYaml .Values | nindent 8 }}`, so we pass the entire map of values to the node helmfile, instead of remapping everytime we add a new value